### ISSUE TITLE
Bug: Calling `browser.waitForAngular` does nothing when blocking proxy is enabled

### ISSUE THREAD

**Issue by sjelin** [angular/protractor#4060]
Rarely would someone directly call `browser.waitForAngular`, but if they did, and blocking proxy were enabled, it wouldn't do anything.  We need to move the current implementation to a private `browser.waitForAngular_` and make a public `browser.waitForAngular` that's just a blank `executeScript` or something.

**Comment by heathkit** [comment]
Good point, waitForAngular should be private (which would be a breaking change).
